.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ on:
   pull_request:
   schedule:
     # Runs "At 00:01" (see https://crontab.guru)
-    - cron: "1 0 * * 0" # every Sunday (relevant for non-Dask TPC-H benchmarks) 
+    - cron: "1 * * * *" # every Sunday (relevant for non-Dask TPC-H benchmarks) 
```

Comment:
This will be reverted in a follow-up, I just want to be able to observe this on `main`.